PurchasesCoreSwift/Attribution/AttributionData.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  AttributionData.swift
+//  PurchasesCoreSwift
+//
+//  Created by Madeline Beyl on 7/7/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+// TODO(post-migration): Make this internal
+@objc(RCAttributionData) public class AttributionData: NSObject {
+
+    @objc public var data: [AnyHashable: Any]
+    @objc public var network: AttributionNetwork
+    @objc public var networkUserId: String?
```

Comment:
The three `var`s can be `let`s 🎉

Reply:
i had them as lets originally and things were working, but then saw that in objc it was readwrite... ?

Reply:
Not surprised at all, the original could have been `readonly` 😄
I think during this migration, if you come across things that are internal API and the access level or mutability is wrong, you should feel free to update it.
I tried it out and like you said, it works. If it didn't work, it just wouldn't compile ❌

Reply:
+1. 
I just double-checked just in case and we don't write to them anywhere outside the class, the `readwrite` was most likely an oversight